src/Build/Evaluation/Evaluator.cs

```diff
@@ -1847,6 +1849,8 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
                 // Combine SDK path with the "project" relative path
                 try
                 {
+                    // TODO: can we distinguish builtin and custom sdkresolvers here?
+                    using var _ = AssemblyLoadsTracker.StartTracking(_evaluationLoggingContext, AssemblyLoadingContext.SdkResolution);
```

Comment:
The only built-in one is `DefaultSdkResolver` so an explicit check for that should suffice.